packages/flutter/lib/src/widgets/reorderable_list.dart

```diff
@@ -557,6 +563,7 @@ class SliverReorderableListState extends State<SliverReorderableList> with Ticke
   @override
   void dispose() {
     _dragInfo?.dispose();
+    _autoScroller!.stopAutoScroll();
```

Comment:
Can't this still be null? Shouldn't it be: 
```suggestion
    _autoScroller?.stopAutoScroll();
```